### PR TITLE
[Merged by Bors] - chore: remove >9 month old deprecations

### PR DESCRIPTION
**Pull request by kim-em**

### CODE REVIEW

File: Mathlib/Analysis/SpecificLimits/Basic.lean [hunk 1]
```diff
@@ -317,18 +285,15 @@ theorem hasSum_geometric_of_lt_one {r : ℝ} (h₁ : 0 ≤ r) (h₂ : r < 1) :
...
   (hasSum_iff_tendsto_nat_of_nonneg (pow_nonneg h₁) _).mpr <| by
     simp_all [neg_inv, geom_sum_eq, div_eq_mul_inv]
-@[deprecated (since := "2024-01-31")] alias hasSum_geometric_of_lt_1 := hasSum_geometric_of_lt_one
 
 theorem summable_geometric_of_lt_one {r : ℝ} (h₁ : 0 ≤ r) (h₂ : r < 1) :
     Summable fun n : ℕ ↦ r ^ n :=
   ⟨_, hasSum_geometric_of_lt_one h₁ h₂⟩
 
-@[deprecated (since := "2024-01-31")]
 alias summable_geometric_of_lt_1 := summable_geometric_of_lt_one
```
Comment (grunweg) [inline Mathlib/Analysis/SpecificLimits/Basic.lean:293]: Please remove this line also.

**Review by grunweg — COMMENTED** [review]
Thanks! Please remove the commented alias also, and fix the build (a one-line fix for the longFile linter).
maintainer delegate

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

File: Mathlib/Analysis/SpecificLimits/Basic.lean (hunk 1 above)
Comment (jcommelin) [inline Mathlib/Analysis/SpecificLimits/Basic.lean:293]:
```suggestion
```

**Comment by grunweg** [comment]
The build still has a linter failure - it will not pass bors as-is. Can you please fix this? Thanks :-)

**Comment by Ruben-VandeVelde** [comment]
maintainer merge

**Comment by jcommelin** [comment]
bors d+

bors d=Ruben-VandeVelde

**Comment by Ruben-VandeVelde** [comment]
bors single on

bors merge

**Pull request closed without merging**